src/index.test.ts

```diff
@@ -74,3 +78,40 @@ test.serial("cleanDatabase() method should remove all items", async t => {
   result = await request(serverUrl, server.getQueries().findAllItems);
   t.assert(result.findAllItems.length === 0, "The item should be gone after calling cleanDatabase() method")
 })
+
+test.serial("setData() should init DB with specified data and replace existing data", async t => {
+  const server = new TestxServer(ITEM_MODEL);
+  
+  await server.start();
+  const serverUrl = server.url()
+
+  await request(serverUrl, CREATE_ITEM);
+  let result = await request(serverUrl, server.getQueries().findAllItems);
+  t.assert(result.findAllItems.length === 1, "Created item should be successfully fetched")
+
+  await server.setData({
+    item: [ {id: 0, title: "foo"}, {id: 1, title: "bar"} ]
+  })
+  result = await request(serverUrl, server.getQueries().findAllItems);
+  t.assert(result.findAllItems.length === 2, "Only items created with setData() method should be fetched")
+  t.assert(result.findAllItems.filter(i => i.title.match(/foo|bar/g).length === 2))
+})
+
+test.serial("getGraphQLSchema() method should produce GQL schema", async t => {
+  const server = new TestxServer(ITEM_MODEL);
+  
+  await server.start()
+  const schema = server.getGraphQlSchema()
+  t.assert(typeof schema === 'string')
+})
```

Comment:
@b1zzu @jhellar  Not sure about this test. Do you think it makes sense to test all expected types and inputs the method should produce?

Reply:
This is enough, in my opinion, but maybe we could also ensure that we have at least n lines (something low like 10) and that the schema complies correctlly (gql doesn't throw execeptions)